packages/core/src/components/table/table/test/expandable-row/table.e2e.ts

```diff
@@ -10,7 +10,7 @@ test.describe('tds-table-expandable-row', () => {
     await expect(tableComponent).toHaveCount(1);
 
     /* check of diff in component screenshot */
-    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0.01 });
```

Comment:
Why the 0.01 value?

Reply:
Nevermind, I saw your comment now!

Reply:
I wrote it under Additional content: 
_I have increased pixel tolerance in table tests as usually they fail when 1% of total number of pixel is off different. It tends to happen in components with dropdowns or some kind of change on click etc, here it is filtering and expandable rows..._

We can check also other ways to improve it, I added tolerance as it is one way to get table tests failing, we can also increase it to higher value too, like 2-5%. There is also `waitFor()` method that can be called items before screenshot is taken, that is another possible improvement too.